.goreleaser.yml

```diff
@@ -30,3 +30,14 @@ checksum:
   name_template: 'checksums.txt'
 snapshot:
   name_template: "{{ .Tag }}-next"
+dockers:
+  -
+    goos: linux
+    goarch: amd64
```

Comment:
I guess having images for other architectures (at least arm64) doesn't hurt.

Reply:
That would mean I have to create multiple different configuration to create and push the docker image. I'm not sure it's worth it.
We could still do it later if someone is asking. But I doubt a bit on that.